samples/client/petstore/swift5/alamofireLibrary/docs/FakeAPI.md

```diff
@@ -228,7 +228,7 @@ For this test, the body for this request much reference a schema named `File`.
 // The following code samples are still beta. For any issue, please report via http://github.com/OpenAPITools/openapi-generator/issues/new
 import PetstoreClient
 
-let body = FileSchemaTestClass(file: File(sourceURI: "sourceURI_example"), files: [File(sourceURI: "sourceURI_example")]) // FileSchemaTestClass | 
+let body = FileSchemaTestClass(file: File(sourceURI: "sourceURI_example"), files: [nil]) // FileSchemaTestClass | 
```

Comment:
@4brunu are we ok with these changes in the code sample in the doc?

Reply:
I tested it locally and the only effect on my project is also in the docs.
So maybe this is an issue that we can address later, and this way at least the generator works.

Reply:
👌